src/app/calendar/page.tsx

```diff
@@ -0,0 +1,7 @@
+export default function Reservation() {
+    return (
+        <main className="flex min-h-screen flex-col items-center justify-between p-24">
```

Comment:
In a lot of places, you repeat the same HTML structure. Ideally, those things go to `layout.tsx` files: https://nextjs.org/docs/app/building-your-application/routing/layouts-and-templates